amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -139,14 +149,30 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
         <main className="column-layout-1">
           <section className="left-panel">
             <div className="section-title title-3">Description</div>
-            <div>
-              { dashboard.description }
-            </div>
+            {
+              hasDescription &&
+              <div>
+                { dashboard.description }
+              </div>
+            }
+            {
+              !hasDescription &&
+              <span>
```

Comment:
nit: Can we reuse the `div` from the `hasDescription &&` block or does this need to be a `span`?

Reply:
`<span>` is probably not needed will remove.